lib/provisioner/worker/plugins/providers/fog_provider/lib/fog_provider/google.rb

```diff
@@ -282,20 +286,10 @@ def delete(inputmap)
         existing_disks.each do |disk|
           log.debug "Issuing delete for disk #{disk.name}"
           begin
-            disk.destroy
-          rescue Fog::Errors::NotFound
-            log.debug 'Disk already deleted'
-          end
-        end
-
-        # confirm all disks deleted
-        existing_disks.each do |disk|
-          begin
-            disk.wait_for(120) { !ready? }
+            disk.destroy(false) # async = false
           rescue Fog::Errors::NotFound
-            log.debug "Disk #{disk.name} no longer found"
+            log.debug 'Disk not found'
```

Comment:
Can we keep the `disk.name` in this output? Otherwise, the output becomes confusing under multiple simultaneous deletes.